pkg/kube_events_manager/types/types.go

```diff
@@ -80,17 +89,29 @@ func (o ObjectAndFilterResult) MarshalJSON() ([]byte, error) {
 
 func (o *ObjectAndFilterResult) RemoveFullObject() {
 	o.Object = nil
-	o.ObjectBytes = 0
+	o.ObjectSize = 0
 	o.Metadata.RemoveObject = true
 }
 
 type ObjectAndFilterResults map[string]*ObjectAndFilterResult
 
 func (a ObjectAndFilterResults) Bytes() (size int64) {
 	for _, o := range a {
-		size += int64(len(o.FilterResult))
-		size += o.ObjectBytes
+		if newSize, overflowed := overflow.Add64(size, int64(o.FilterResultSize)); overflowed {
+			size = math.MaxInt64
```

Comment:
In case of overflow, we lose precision. It looks like we need to add some warning here or even to return an error. WDYT?

Reply:
Although exabyte-scaled binding contexts are possible, I think the shell-operator should fail before counting their size =)

Reply:
I've obliterated the overflow test. Why? Well, we are losing even more precision when we export metrics by converting them to float64. I'd rework this calculation in v2 and skip it completely for now.

![image](https://user-images.githubusercontent.com/950313/113116794-754f5000-9216-11eb-8ffc-cd1008ab47df.png)